corehq/apps/es/tests/__init__.py

```diff
@@ -4,3 +4,9 @@
 from .test_filters import *
 from .test_queries import *
 from .test_case_search_es import *
+from corehq.apps.es.case_search import flatten_result
+
+
+__test__ = {
+    'flatten_result': flatten_result
+}
```

Comment:
Oh, not at all. I just saw your example in the docstring, and thought, "That example looks just like a doctest. I should just put some `>>>`s in there". But testing exactly the same thing twice seems like a waste, and I think it's good to have the tests together.